### PR TITLE
[SQL] Adjust multiplicative augments to behave properly

### PR DESCRIPTION
**Pull request by MowFord**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?

Augments have a particular mod they enact (or multiple mods). Each mod is given a power and a multiplier. I think whoever implemented augments intended for power to always be 1 or -1 when multiplier was nonzero.

Finding augments via this query:
`SELECT * FROM augments WHERE multiplier <> 0 AND ABS(VALUE) <> 1`

we can see various mismatches between what the client shows and what mods get applied:

image

image

image

This PR attempts to correct them all in one fell swoop. I am aware of an over-arching issue with augments not being packet-correct and thus sometimes giving incorrect display when augmentid/power aren't within certain parameters (i.e. [this issue](https://github.com/LandSandBoat/server/issues/2772)), but I don't believe that is related to this particular PR's resolution

Basically, the intent of this PR is to find all augments that the client interprets as counting by multiples other than 1 and fixing the SQL side to make the CPP side work in the expected way.

## Steps to test these changes

after changes:
image
etc

specifically, you test by giving yourself an item with the particular augment/power and see that the game client reflects the underlying mods being applied (I like to use bronze dagger myself):
- `!additem bronze_dagger 1 78 0`
  - image
- `!additem bronze_dagger 1 78 2`
  - image
- `!additem bronze_dagger 1 83 1`
  - image
etc, etc

### CODE REVIEW

**Comment by MowFord** [comment]
Went through and confirmed each augment id, but thought i'd show the pet ones in addition to the above screenshots since they're somewhat unique:
image
image

File: sql/augments.sql [hunk 1]
```diff
@@ -1427,8 +1427,8 @@ INSERT INTO `augments` VALUES (1151,0,0,0,0,0);
 INSERT INTO `augments` VALUES (1152,0,1,10,0,0); -- DEF +10
 INSERT INTO `augments` VALUES (1153,0,68,3,0,0); -- Evasion +3
 INSERT INTO `augments` VALUES (1154,0,31,3,0,0); -- Mag. Evasion +3
-INSERT INTO `augments` VALUES (1155,100,161,-2,0,0); -- Physical Damage Taken -2%
-INSERT INTO `augments` VALUES (1156,100,163,-2,0,0); -- Magic Damage Taken -2%
+INSERT INTO `augments` VALUES (1155,200,161,-1,0,0); -- Physical Damage Taken -2%
+INSERT INTO `augments` VALUES (1156,200,163,-1,0,0); -- Magic Damage Taken -2%
```
Comment (TeoTwawki) [inline sql/augments.sql:1431]: can you take a look at aug ID 71 also? thanks.

Comment (MowFord) [reply sql/augments.sql:1431]: image

🔨

Comment (MowFord) [reply sql/augments.sql:1431]: found those 3 more augments and fixed to match the rest of what this PR is doing. Essentially some augments do have a base value then increment by 1, but _every_ mod that we (lsb) chooses to have some other base on the values has to use mutlipliers even if they increment "by one" as in these extra 3.

TL;DR - the 3 above had this behavior:
image

Comment (TeoTwawki) [reply sql/augments.sql:1431]: thanks.

**Review by TeoTwawki — APPROVED** [review]

**Review by Xaver-DaRed — APPROVED** [review]

**Review by zach2good — APPROVED** [review]
